scripts/init-sg721.ts

```diff
@@ -21,14 +21,13 @@ async function main() {
   const collectionMsg = {
```

Comment:
Since the minter creates the sg721 I don't think we need this. Might be too confusing for creators right now.

Reply:
Ok, will just delete this script then.